Updates file format details for Data Visualizer documentation (#193819)
elastic/kibana: docs/user/ml/index.asciidoc

[role="screenshot"]
image::user/ml/images/ml-data-visualizer-sample.png[{data-viz} for sample flight data]

> You can upload different file formats for analysis with the *{data-viz}*. The following file types are 
> supported up to 500 MB: CSV, TSV, NDJSON, and log files. 
> The following formats are supported up to 60 MB: PDF, Microsoft Office files (Word, Excel, PowerPoint), Plain Text (TXT), Rich Text (RTF), and Open Document Format (ODF).

Reviewer: ```suggestion
You can upload different file formats for analysis with the *{data-viz}*.

Files supported up to 500 MB include:
* CSV
* TSV
* NDJSON
* Log files

Files supported up to 60 MB include:
* PDF
* Microsoft Office files (Word, Excel, PowerPoint)
* Plain Text (TXT)
* Rich Text (RTF)
* Open Document Format (ODF)
```

> The *{data-viz}* identifies the file format and field mappings, and you can optionally import the data into an {es} index. To change the default file size limit, see <<kibana-general-settings, fileUpload:maxFileSize in advanced settings>>.

Reviewer: ```suggestion
The *{data-viz}* identifies the file format and field mappings, and you can import the data into an {es} index. To change the default file size limit, see <<kibana-general-settings, `fileUpload:maxFileSize`>> in advanced settings.
```

nit changes to existing text :)
